.github/workflows/release-connect-v9-staging.yml

```diff
@@ -50,7 +50,7 @@ jobs:
   # This job deploys to staging-connect.trezor.io/9.x.x
   deploy-staging-semantic-version:
     needs: [extract-version, check-version-match]
-    environment: production
+    environment: staging-connect
```

Comment:
Cool, so if I understand well this new environment is not going to required approval from 2 authorized members right?

Reply:
Yes we can better manage the rules for different environments :)